express-post-json/server.js

```diff
@@ -0,0 +1,28 @@
+import express from 'express';
+
+const app = express();
+
+let nextId = 1;
+const grades = {};
+
+app.get('/api/grades', (req, res) => {
+  const gradeArray = [];
+  for (const key in grades) {
+    gradeArray.push(grades[key]);
+  }
+  res.json(gradeArray);
+});
+
+app.use(express.json());
```

Comment:
Usually this would go up at the top. But as long as it comes ahead of the endpoints that need it, it's OK to have it anywhere.